deps/wazuh_testing/wazuh_testing/qa_docs/doc_generator.py

```diff
@@ -105,11 +118,22 @@ def dump_output(self, content, doc_path):
             - "doc_path (string): The path where the information should be dumped."
         """
         if not os.path.exists(os.path.dirname(doc_path)):
+            DocGenerator.LOGGER.warning('Creating documentation folder')
             os.makedirs(os.path.dirname(doc_path))
-        with open(doc_path + ".json", "w+") as outfile:
-            outfile.write(json.dumps(content, indent=4))
-        with open(doc_path + ".yaml", "w+") as outfile:
-            outfile.write(yaml.dump(content))
+
+        try:
+            DocGenerator.LOGGER.debug(f"Writing {doc_path}.json")
+            with open(doc_path + ".json", "w+") as outfile:
```

Comment:
Perhaps `out_file` would be better (improve readability to separate the words that form the variable name with underscores).
```suggestion
            with open(doc_path + ".json", "w+") as out_file:
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523